### ISSUE TITLE
Shadow not working on Stack and Box

### ISSUE THREAD

**Issue by omar-bear** [BearStudio/react-native-ficus-ui#23]
## Description
shadow props not working on `Stack` and `Box` component

```
<Stack
      spacing="lg"
      h={80}
      pt={10}
      w="100%"
      flexDirection="row"
      shadow="lg"
      alignItems="center"
      bg="white"
      px="xl"
      borderBottomLeftRadius={32}
      borderBottomRightRadius={24}
    />
```

**Comment by omar-bear** [comment]
it works but I have to add shadowColor to make it work, i think we have to add a default shadowColor value

**Comment by ntorionbearstudio** [comment]
Fixed on version 1.0.0
